packages/core/src/index.ts

```diff
@@ -7,8 +7,8 @@
 export {Component, ProviderMap} from './component';
 
 // loopback dependencies
-export {inject, Context} from '@loopback/context';
-export {Server} from './server';
+export * from '@loopback/context';
```

Comment:
Any reason for the switch to `*`?

Reply:
Typical applications need more than just `inject` and `Context`. When I revert this change, I see the following compilation errors:

```
examples/todo/src/application.ts(6,28): error TS2305: 
  Module '"packages/core/index"' has no exported member 'Provider'.

examples/todo/src/application.ts(6,38): error TS2305: 
  Module '"packages/core/index"' has no exported member 'Constructor'.

examples/todo/src/services/geocoder.service.ts(7,17): error TS2305: 
  Module '"packages/core/index"' has no exported member 'Provider'.
```